Main_for_ODBS.py

```diff
@@ -0,0 +1,150 @@
+from Nika_modules import FASTQ_tools, dna_rna_tools, amino_acids_tools, read_fastq_file, write_output
+from Nika_modules.amino_acids_tools import single_letter_alphabet, three_letter_alphabet
+
+
+def filter_fastq(input_path, gc_bounds=(0, 100), length_bounds=(0, 2**32), quality_threshold=0, output_filename=None):
+    """
+    Takes fastq sequences(dict), where is the key - sequence name(str), and the value is a tuple of two strings:
+    sequence and quality. Returns the dictionary, consisting only of those sequences that passed all the conditions.
+    :param input_path: str. Path to fastq file.
+    :param output_filename: str. Name of output file.
+    :param gc_bounds: int, float or tuple. If it's an int or float, then the function discards reads
+    with a GC composition lower than this value.
+    :param length_bounds: int, float or tuple. If it's an int or float, then the function discards reads
+    with a length lower than this value.
+    :param quality_threshold: int. All reads with quality below the value will be discarded.
+    :return: dict.
+    """
+    filtered_seqs = {}
+    fastq_file = read_fastq_file.read_fastq(input_path)
```

Comment:
Это же не файл уже, можно как нибудь типа
```suggestion
    fastq_data = read_fastq_file.read_fastq(input_path)
```